riak/tests/test_timeseries_ttb.py

```diff
@@ -130,6 +132,24 @@ class TimeseriesTtbTests(IntegrationTestBase, unittest.TestCase):
     def setUpClass(cls):
         super(TimeseriesTtbTests, cls).setUpClass()
 
+    def test_query_that_creates_table_using_interpolation(self):
+        table = ''.join(
+            [random.choice(string.ascii_letters + string.digits)
+                for n in range(32)])
+        query = """CREATE TABLE test-{table} (
+            geohash varchar not null,
+            user varchar not null,
+            time timestamp not null,
+            weather varchar not null,
+            temperature double,
+            PRIMARY KEY((geohash, user, quantum(time, 15, m)),
+                geohash, user, time))
+        """
+        ts_obj = self.client.ts_query(table, query)
+        self.assertIsNotNone(ts_obj)
+        self.assertFalse(hasattr(ts_obj, 'ts_cols'))
+        self.assertIsNone(ts_obj.rows)
+
     def test_query_that_returns_table_description(self):
         fmt = 'DESCRIBE {table}'
```

Comment:
@javajolt - how did this `DESCRIBE` test not find this? Doesn't `DESCRIBE` return `tsqueryresp` here:

https://github.com/basho/riak_kv/blob/riak_ts-develop/src/riak_kv_ts_svc.erl#L144-L145